docs/content/en/docs/crd-ref/_index.md

```diff
@@ -5,23 +5,44 @@ weight: 100
 hidechildren: false # this flag hides all sub-pages in the sidebar-multicard.html
 ---
 
-This section provides comprehensive reference information about the
+This section provides comprehensive reference information about all
 [Custom Resource Definitions (CRDs)](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/)
 that are defined for the Keptn Lifecycle Toolkit.
-
-> **Note**
-This section is under development.
-Information that is published here has been reviewed for technical accuracy
-but the format and content is still evolving.
-We welcome your input!**
+This section is auto-generated from the source code.
 
 Each CRD is an object of an API library.
 Keptn APIs follow the Kubernetes API versioning scheme.
 and are themselves composed of objects and sub-objects.
-By introducing CRDs, Keptn is extending the base Kubernetes API with new objects and functionality.
+The Keptn CRDs extend the base Kubernetes API
+with new objects and functionality.
 Keptn APIs follow API versioning conventions recommended by Kubernetes.
 
-For more information, see the Kubernetes documentation:
+Keptn generates most of the resources it needs
+without requiring manual input.
+[Manifest CRD Reference](../yaml-crd-ref)
+contains reference pages for the few manifests
+that must be populated manually.
+
+Use `kubectl` to inspect the current contents of any Keptn resource:
+
+1. List all resources of the specified type within a certain namespace.
+   For example, to list all the `KeptnApp` resources
+   in the `namespace1` namespace, the command is:
+
+   ```shell
+   kubectl get keptnapp -n namespace1
```

Comment:
For better understanding, I would use the plural forms when `kubectl get`ting multiple resources at once
```suggestion
   kubectl get keptnapps -n namespace1
```